evap/results/tests/test_exporters.py

```diff
@@ -367,11 +367,40 @@ def test_exclude_used_but_unanswered_questionnaires(self):
         cache_results(evaluation)
 
         sheet = self.get_export_sheet(evaluation.course.semester, degree, [evaluation.course.type.id])
-        self.assertEqual(sheet.row_values(4)[0], used_questionnaire.name)
+        self.assertEqual(sheet.row_values(4)[0], used_questionnaire.public_name)
         self.assertEqual(sheet.row_values(5)[0], used_question.text)
         self.assertNotIn(unused_questionnaire.name, sheet.col_values(0))
         self.assertNotIn(unused_question.text, sheet.col_values(0))
 
+    def test_show_display_name_of_questionnaires(self):
```

Comment:
Since we are now always showing the public name, could this test be removed?

Reply:
Right – in case the wrong questionnaire title is displayed, it would be pretty clear that this is causing the all the other tests to fail, because all the assertion statements asserting the name would be failing. 
Thus I'll remove this test if that's okay